payments-core/src/main/java/com/stripe/android/view/CardWidgetViewModel.kt

```diff
@@ -97,6 +99,7 @@ internal class CardWidgetViewModel(
 
 internal fun View.doWithCardWidgetViewModel(
     viewModelStoreOwner: ViewModelStoreOwner? = null,
+    application: Context? = null,
```

Comment:
We shouldn't need to pass this, you can do `view.getContext().getApplicationContext()` -- this is java syntax, compose be fancier.